content/SCALE/SCALE22.12.md

```diff
@@ -450,7 +450,8 @@ Additional feature in future Bluefin releases:
 
 | Seen In | Key | Summary | Workaround | Resolved In |
 |---------|-----|---------|------------|-------------|
-| 22.12-BETA.1 | <a href="https://ixsystems.atlassian.net/browse/NAS-117940" target="_blank">NAS-117940</a> | Implements temporary fix for the return from `glfs_open()` to honor `O_DIRECTORY` flag | Pertains to an internal issue in Samba. After gluserfs is fixed with a permanent solution to this issue happens, this temporary fix will be reverted. | Targeted 22.12 |
+| 22.12-BETA.1  |  | Upgrading from 22.02.4 to 22.12-BETA.1 is known to not work. | Workaround is to either upgrade from a version before 22.02.4 or to upgrade to 22.12-BETA.2. | 22.12-BETA.2 |
```

Comment:
`upgrade to 22.12-BETA.2`, maybe note that BETA.2 is not out yet... "wait to upgrade to 22.12-BETA.2 once it is released" or something

Reply:
I'm in the BETA.2 rel note process now so I'll make sure this is in that as well as in the PR. I'll fix this to targeted for the fixed in.